tests/testing_utils.py

```diff
@@ -34,15 +34,16 @@ class SceneTester:
     def __init__(self, scene_object, module_tested, caching_needed=False):
         # Disable the the logs, (--quiet is broken) TODO
         logging.disable(logging.CRITICAL)
-        self.path_tests_medias_cache = os.path.join('tests_cache', module_tested)
-        self.path_tests_data = os.path.join('tests_data', module_tested)
+        self.path_tests_medias_cache = os.path.join('tests', 'tests_cache', module_tested)
+        self.path_tests_data = os.path.join('tests', 'tests_data', module_tested)
 
         if caching_needed:
             config['text_dir'] = os.path.join(
                 self.path_tests_medias_cache, scene_object.__name__, 'Text')
             config['tex_dir'] = os.path.join(
                 self.path_tests_medias_cache, scene_object.__name__, 'Tex')
 
+        file_writer_config['skip_animations'] = True
         config['pixel_height'] = 480
```

Comment:
Remind me why these are being set here? The whole point of having a `tests/manim.cfg` was so that this wasn't necessary. Not asking for a change, just curious.

Reply:
I had choice. I agree with you, this should live in a `manim.cfg`.

But this is impossible at the moment, see #184

Reply:
Yeah ok that's what I thought. In the interest of merging this PR, let's do this for now and make a note to come back to this until #184 is resolved.

Reply:
Excalty what I planend to do :)